pkg/telemetrymetadata/metadata.go

```diff
@@ -1164,6 +1172,10 @@ func (t *telemetryMetaStore) getLogFieldValues(ctx context.Context, fieldValueSe
 		limit = 50
 	}
 
+	if strings.HasPrefix(fieldValueSelector.Name, telemetrytypes.BodyJSONStringSearchPrefix) {
+		return t.ListJSONValues(ctx, fieldValueSelector.Name, limit)
+	}
```

Comment:
This is going to be very expensive on production workloads, we shouldn't use main table for value suggestions.

Reply:
Yes, we had the discussions to remove from the main flow